catroidTest/src/org/catrobat/catroid/test/physics/PhysicsSpriteCloneTest.java

```diff
@@ -188,7 +187,7 @@ public void testSpriteClonePhysicsLookAndPhysicsObject() {
 		sprite.addScript(startScript);
 
 		PhysicsWorld physicsWorld = project.getPhysicsWorld();
-		sprite.look = new PhysicsLook(sprite, physicsWorld);
+		sprite.look = new Look(sprite);
```

Comment:
If you clone a sprite, all it's data is cloned too, including the look data values.

So, if you create a sprite with a set transparency to 50% brick, start and leave the program, copy the sprite, go into that newly copied sprite and compute the transparency, it displays 50.

However, the method mentioned in my previous comment is missing some values to copy (like x, y, etc.), and tests. I'll write a jira ticket for that.